### PR TITLE
[iOS] 12578 moved demo app to cocoapods

### PR DESCRIPTION
**Pull request by adam-govan**
@Sergio-Mira / @dominicmarmionswrve

### CODE REVIEW

File: SwrveDemo/Podfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+target 'SwrveDemoFramework' do
+pod 'SwrveSDK', '~> 4.4.0'
```
Comment (Sergio-Mira) [inline SwrveDemo/Podfile]: target local path

Comment (adam-govan) [inline SwrveDemo/Podfile]: the point is to not target local path so they're using the same tools as we are.

Comment (Sergio-Mira) [inline SwrveDemo/Podfile]: It'll always be one version behind and we can't show new features on the demo then. How about having it local and commented that is the that line but pointing to the future 4.5.0 pod?

Comment (adam-govan) [inline SwrveDemo/Podfile]: Sure 😄  👍

File: SwrveDemo/Podfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+target 'SwrveDemoFramework' do
+    pod 'SwrveSDKCommon', :path => '../'
+    pod 'SwrveConversationSDK', :path => '../'
+    pod 'SwrveSDK', :path => '../'
+    #pod 'SwrveSDK', '~> 4.5.0'
```
Comment (dominicmarmionswrve) [inline SwrveDemo/Podfile:8]: Perhaps add a comment to above line, explaining to comment out above 3 lines and uncomment below to use remote cocoapod....

**Comment by dominicmarmionswrve** [comment]
LGTM! 🚀

File: SwrveDemo/Podfile (hunk 2 above)
Comment (adam-govan) [inline SwrveDemo/Podfile:8]: that's a good idea actually

**Pull request merged**